SimCalorimetry/HGCalSimProducers/src/HGCDigitizer.cc

```diff
@@ -330,392 +319,386 @@ void HGCDigitizer::finalizeEvent(edm::Event& e, edm::EventSetup const& es, CLHEP
     bool orderChanged = false;
-    if(itime == 9){
-      if(hitRefs_bx0[id].empty()){
-	hitRefs_bx0[id].emplace_back(charge, tof);
-      }
-      else if(tof <= hitRefs_bx0[id].back().second){
-	std::vector<std::pair<float, float> >::iterator findPos =
-	  std::upper_bound(hitRefs_bx0[id].begin(), hitRefs_bx0[id].end(), std::pair<float, float>(0.f,tof),
-			   [](const auto& i, const auto& j){return i.second < j.second;});
-
-	std::vector<std::pair<float, float> >::iterator insertedPos =
-	  hitRefs_bx0[id].insert(findPos, (findPos == hitRefs_bx0[id].begin()) ?
-				 std::pair<float, float>(charge,tof) : std::pair<float, float>((findPos-1)->first+charge,tof));
-
-	for(std::vector<std::pair<float, float> >::iterator step = insertedPos+1; step != hitRefs_bx0[id].end(); ++step){
-	  step->first += charge;
-	  if(step->first > tdcForToAOnset[waferThickness-1] && step->second != hitRefs_bx0[id].back().second){
-	    hitRefs_bx0[id].resize(std::upper_bound(hitRefs_bx0[id].begin(), hitRefs_bx0[id].end(), std::pair<float, float>(0.f,step->second),
-						    [](const auto& i, const auto& j){return i.second < j.second;}) - hitRefs_bx0[id].begin());
-	    for(auto stepEnd = step+1; stepEnd != hitRefs_bx0[id].end(); ++stepEnd) stepEnd->first += charge;
-	    break;
-	  }
-	}
-	orderChanged = true;
-      }
-      else{
-        if(hitRefs_bx0[id].back().first <= tdcForToAOnset[waferThickness-1]){
-          hitRefs_bx0[id].emplace_back(hitRefs_bx0[id].back().first+charge, tof);
+    if (itime == 9) {
+      if (hitRefs_bx0[id].empty()) {
+        hitRefs_bx0[id].emplace_back(charge, tof);
+      } else if (tof <= hitRefs_bx0[id].back().second) {
+        std::vector<std::pair<float, float>>::iterator findPos =
+            std::upper_bound(hitRefs_bx0[id].begin(),
+                             hitRefs_bx0[id].end(),
+                             std::pair<float, float>(0.f, tof),
+                             [](const auto& i, const auto& j) { return i.second < j.second; });
+
+        std::vector<std::pair<float, float>>::iterator insertedPos = hitRefs_bx0[id].insert(
+            findPos,
+            (findPos == hitRefs_bx0[id].begin()) ? std::pair<float, float>(charge, tof)
+                                                 : std::pair<float, float>((findPos - 1)->first + charge, tof));
+
+        for (std::vector<std::pair<float, float>>::iterator step = insertedPos + 1; step != hitRefs_bx0[id].end();
+             ++step) {
+          step->first += charge;
+          if (step->first > tdcForToAOnset[waferThickness - 1] && step->second != hitRefs_bx0[id].back().second) {
+            hitRefs_bx0[id].resize(std::upper_bound(hitRefs_bx0[id].begin(),
+                                                    hitRefs_bx0[id].end(),
+                                                    std::pair<float, float>(0.f, step->second),
+                                                    [](const auto& i, const auto& j) { return i.second < j.second; }) -
```

Comment:
this looks very awkward to me, but I think it is technically in agreement with the indentation rules in the code format settings

Reply:
@kpedro88 I've seen several similar cases. They look aesthetically odd, but the more I see them the more I am convinced the logic behind the splitting might be practically helpful in distinguishing blocks of the expression